### PR TITLE
Vidazoo Adapter: added session management

### PR DESCRIPTION
**Pull request by uditalias**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- Server request method changed from GET to POST
- Added session management
- Added tests

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
devs@vidazoo.com

- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/vidazooBidAdapter.js [hunk 1]
```diff
@@ -86,8 +93,22 @@ function interpretResponse(serverResponse, request) {
   }
 }
 
+function setLocalStorage(key, value) {
```
Comment (mkendall07) [inline modules/vidazooBidAdapter.js]: please use `utils.setDataInLocalStorage` instead

File: modules/vidazooBidAdapter.js [hunk 2]
```diff
@@ -86,8 +93,22 @@ function interpretResponse(serverResponse, request) {
...
+  try {
+    window.localStorage.setItem(key, value);
+  } catch (ex) {
+  }
+}
+
+function getLocalStorage(key) {
```
Comment (mkendall07) [inline modules/vidazooBidAdapter.js]: please use `utils.getDataFromLocalStorage` instead.

**Comment by uditalias** [comment]
Thanks, @mkendall07 
We just changed it now

**Comment by uditalias** [comment]
waiting for review

File: modules/vidazooBidAdapter.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import * as utils from '../src/utils';
-import {registerBidder} from '../src/adapters/bidderFactory';
-import {BANNER} from '../src/mediaTypes';
-export const URL = 'https://prebid.cootlogix.com';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
+export const URL = '//prebid.cootlogix.com';
```
Comment (mkendall07) [inline modules/vidazooBidAdapter.js]: please keep hardcoded HTTPS, this is a requirement now.

**Comment by uditalias** [comment]
Waiting for review

**Pull request closed without merging**